movingpandas/tests/test_geometry_utils.py

```diff
@@ -64,4 +64,5 @@ def test_anglular_difference_twonegative(self):
         assert angular_difference(-200, -160) == 40
 
     def test_mrr_diagonal(self):
-        assert mrr_diagonal([Point(0, 0), Point(0, 2), Point(2, 0), Point(2, 2)]) == sqrt(8)
+        """Distance between NYC, NY USA and Los Angeles, CA USA is 3944411.0951634306 meters"""
```

Comment:
Please create a dedicated test for the geodesic distance function and revert the change to this test.